doc/ceph-volume/lvm/encryption.rst

```diff
@@ -4,42 +4,41 @@ Encryption
 ==========
 
 Logical volumes can be encrypted using ``dmcrypt`` by specifying the
-``--dmcrypt`` flag when creating OSDs. Encryption can be done in different ways,
-specially with LVM. ``ceph-volume`` is somewhat opinionated with the way it
-sets up encryption with logical volumes so that the process is consistent and
+``--dmcrypt`` flag when creating OSDs. When using LVM, logical volumes can be
+encrypted in different ways. ``ceph-volume`` does not offer as many options as
+LVM does, but it encrypts logical volumes in a way that  is consistent and
 robust.
 
-In this case, ``ceph-volume lvm`` follows these constraints:
+In this case, ``ceph-volume lvm`` follows this constraint:
 
-* Logical Volumes are encrypted, while their underlying PVs (physical volumes)
-  aren't
-* Non-LVM devices like partitions are also encrypted with the same OSD key
+* Non-LVM devices (such as partitions) are encrypted with the same OSD key.
 
 
 LUKS
 ----
-There are currently two versions of LUKS, 1 and 2. Version 2 is a bit easier
-to implement but not widely available in all distros Ceph supports. 
+There are currently two versions of LUKS, 1 and 2. Version 2 is a bit easier to
+implement but not widely available in all Linux distributions supported by
+Ceph. 
 
-.. note:: Version 1 of LUKS is just referenced as "LUKS" whereas version 2 is
-          referred to as LUKS2
+.. note:: Version 1 of LUKS is referred to in this documentation as "LUKS".
+   Version 2 is of LUKS is referred to in this documentation as "LUKS2".
 
 
 LUKS on LVM
 -----------
-Encryption is done on top of existing logical volumes (unlike encrypting the
-physical device). Any single logical volume can be encrypted while other
-volumes can remain unencrypted. This method also allows for flexible logical
+Encryption is done on top of existing logical volumes (this is not the same as
+encrypting the physical device). Any single logical volume can be encrypted,
+leaving other volumes unencrypted. This method also allows for flexible logical
 volume setups, since encryption will happen once the LV is created.
```

Comment:
I would phrase this as saying that encryption is done at the `logical volume level` FWIW